README.md

```diff
@@ -30,45 +34,70 @@ log.warn('My Warning');
 log.error('My Error');
 log.critical('My Critical Error');
 ```
+
 ### Formatting
 
 Using util.format(...)
+
 - %s - String.
 - %d - Number (both integer and float).
 - %j - JSON. Replaced with the string '[Circular]' if the argument contains circular references.
 - %% - single percent sign ('%'). This does not consume an argument.
+
 ```javascript
-log.info('My Formatted %s', 'Message')
->> 'My Formatted Message'
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
-log.info('My Formatted %d', 123)
->> 'My Formatted 123'
+log.info('My Formatted %s', 'Message') >> 'My Formatted Message';
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
+log.info('My Formatted %d', 123) >> 'My Formatted 123';
```

Comment:
Was this changed with the formatted? Before it displayed the `>>` on a new line, showing it as console output. Not it's less clear, and looks like bitwise shift.

Reply:
Was due to prettier, fixed.